app/index.html

```diff
@@ -3,7 +3,7 @@
     <head>
         <meta charset="utf-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-        <title></title>
+        <title>The Foreman/Katello review dashboard</title>
```

Comment:
I'd be OK with this just saying 'Foreman' given Katello is part of the ecosystem